SwordImportExportPlugin.inc.php

```diff
@@ -191,18 +194,22 @@ public function display($args, $request) {
 
 				$errors = array();
 				// select at least one article
-				$articleIds = $request->getUserVar('articleId');
-				if (empty($articleIds)) {
+				$submissionIds = $request->getUserVar('submissionId');
+				if (is_null($submissionIds)) {
+					$submissionIds = $request->getUserVar('selectedSubmissions');
+				}
+
+				if (empty($submissionIds)) {
 					$errors[] = array(
 						'title' => __('plugins.importexport.sword.requiredFieldErrorTitle'),
 						'message' => __('plugins.importexport.sword.requiredFieldErrorMessage'),
 					);
 				}
 				else {
-					foreach ($articleIds as $articleId) {
-						$publishedArticle = $publishedArticleDao->getByArticleId($articleId);
+					foreach ($submissionIds as $submissionId) {
+						$submission = $submissionDao->getById($submissionId);
```

Comment:
I believe a check is missing here to ensure that the specified submission is in the current journal.

Reply:
Double-checking, @quoideneuf -- is this a potential issue?

Reply:
(Don't forget this one!)

Reply:
@asmecher 👇 isn't that what's needed here?

Reply:
Ah, thanks, the quick view of the PR wasn't showing that.